### ISSUE TITLE
Updated Terms of Service and Privacy Policy

### ISSUE THREAD

**Issue by piamancini** [opencollective/opencollective#322]
Please carefully read our updated [Terms of Service](https://github.com/OpenCollective/OpenCollective/blob/master/TOS.md) and [Privacy Policy](https://github.com/OpenCollective/OpenCollective/blob/master/PRIVACY-POLICY.MD)

Transparency is a core value of Open Collective. And legal documents (with legalese) are often opposite of transparent - they are hard to parse and hard to follow, unless you are a lawyer. We have tried our best to draft them in plain terms and added clarifications wherever we felt it was necessary.

If you have questions or suggestions, please comment below. Open Collective is a community and the rules that organize it are always open to a conversation. 

With <3,

**Comment by tanuk** [comment]
Suggestion: provide an overview of what was changed and why.

**Comment by piamancini** [comment]
thanks @tanuk, also @boneskull raised the same point. 
The Privacy Policy is a first so no changes and the previous ToS where a very short tl;dr that I am pasting below. I will go through them to mark specific changes but for the moment I wanted to at least share them here

OLD Terms of Service: 
TLDR; Open Collective takes 10% + credit card fees (usually 3% + $0.30/transaction) of the money raised by the collective for managing their bookkeeping, taxes, and the admin of reimbursing their expenses. If an appropriate organization, such as Ruby Together, or ie. a 501c3 wants to be the collective’s fiscal sponsor and manage these, then OpenCollective’s fees are reduced to 5% + credit card fees. When you have an expense (could be an invoice from a freelancer), you will need to submit it with a PDF or a picture of the receipt using the app and you will be reimbursed on your PayPal account. All content and intellectual property is always yours. You should give truthful information about your activities, we are not responsible for it. 

Definitions

The Collective: The Collective is the group the uses the Platform to receive recurring funding to carry out activities to support their mission, and submits expenses for those activities for reimbursement, transparently shown in their public page. 

The Platform: The Platform is the software provided by OpenCollective.Inc and it consists - insofar and not exclusively - of a backend to manage expenses, donations and membership and a public page that displays transparently all transactions carried out by the Collective. 

The Host: The host is the organization or individual that acts as the Collective’s fiscal sponsor, receives funds in the name of the Collective and reimburses the Collective’s members. The host can choose to receive a % on top of platform fees. 

Terms
The Host will provide the Collective with space in its legal entity in the form of fiscal sponsorship and the Collective wishes to have the Host act as its fiscal sponsor. The Collective will provide a clear mission statement for their group and a description of their activities.  
In support of the Collective the Host will receive membership fees and donations on behalf of the Collective and will grant all amount through the Open Collective App (minus administrative charges and expenses, see fees below) to the Open Collective.
Responsibilities of the Host: the Host is responsible to store the money collected via the Platform and disburse the net income (see fees below) for the purposes of the Collective. It is also responsible to report in their books all the transactions that have been done on behalf of the Collective.
Responsibilities of the Collective: the Collective is responsible to provide contact information of each member that will be able to submit expenses against the available funds in the Collective. Contact information include, first name, last name, date of birth, address and a picture of their ID card or passport or driving license. The collective is also responsible to keep that information up to date and communicate to the Host any change. The Collective must clearly communicate to the Host any updates of their mission statement.
Responsibilities of the members of the Collective: each member of the collective is responsible to submit expenses with a picture or PDF of the receipt. They are also responsible to keep their personal information up to date (first name, last name, address, date of birth and photo of their ID card, Passport or Driving License, and personal PayPal emails where they will get reimbursed)
The Collective may solicit contributions from its members and donations. The Collective is free to specify the minimal amount of donations or the amount of a membership fee.
Ownership of intellectual property: Any and all tangible or intangible property, including intellectual property belong to the Collective unless explicitly stated different by the Host Organization. OpenCollective when being the Host acknowledges and agrees that any trademarks, trade names, artwork, designs, logos, copy and all other intellectual property provided to the Host by the Collective belong to the Collective. The Host can use the Collective's public page and its contents for promotion purposes.
Limitations: The Collective can only contract with freelancers and will not have the ability to hire employees. The collective can’t get into any agreement that will engage the Host’s legal responsibility
Fees: Open Collective takes 10% + credit card fees (usually 3% + $0.30/transaction) of the money raised by the collective for managing their bookkeeping, taxes, and the admin of reimbursing their expenses. If an appropriate organization, such as Ruby Together, or ie. a 501c3 wants to be the collective’s fiscal sponsor and manage these, then OpenCollective’s fees are reduced to 5% + credit card fees. 

Disclaimer 
These Terms of Services are a draft and are subject to review. We will inform all Collectives about future changes.

**Comment by piamancini** [comment]
New TOS and PP are published already
